### ISSUE TITLE
[A11y 8/25] 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-item#23]
Across screen readers: this is mostly fine, the only thing is that the little star icons in the bottom left avatar with text and icon and the little chat icons that appear to the right of the names within avatar with text and icon and control with text and icon are not accessible with the screen reader. Do you think these convey important info? Should the person be able to know that there is a star there? If so, we should make them accessible with labels.
